samples/grpc-server-aot/src/test/resources/application-ssl.properties

```diff
@@ -0,0 +1,5 @@
+spring.grpc.server.ssl.bundle=ssltest
```

Comment:
I don't think we need either of these ssl related files either if we drop the other redundant ITs